src/stores/root.ts

```diff
@@ -250,6 +250,13 @@ export class RootStore {
 					spotPriceDestDenom: DenomHelper.ibcDenom([{ portId: 'transfer', channelId: 'channel-0' }], 'uatom'),
 					destCoinId: 'cosmos',
 				},
+				{
+					alternativeCoinId: 'pool:udig',
+					poolId: '621',
```

Comment:
Pool 620 is also `dig`: https://app.osmosis.zone/pool/620

Reply:
Seems like an abandoned pool that was created by accident with a 3% swap fee

Reply:
For sure. I assumed all pools needed identified.